src/Language/PureScript/Sugar/Names/Imports.hs

```diff
@@ -32,10 +32,9 @@ findImports
   -> M.Map ModuleName [ImportDef]
 findImports = foldr go M.empty
   where
-  go (ImportDeclaration (pos, _) mn typ qual) result =
-    let imp = (pos, typ, qual)
-    in M.insert mn (maybe [imp] (imp :) (mn `M.lookup` result)) result
-  go _ result = result
+  go (ImportDeclaration (pos, _) mn typ qual) =
+    M.alter (return . ((pos, typ, qual) :) . fromMaybe []) mn
+  go _ = id
```

Comment:
Am I correct in thinking the change in this file is just a refactor which shouldn't change behaviour?

Reply:
Yeah. I think Gary's advice from a different pull request was still in mind when I made this change. It's meant to be just a simple refactor like you say, but maybe I should undo this patch so everything here is related directly to linting. I don't mind either way.

Reply:
Happy to keep this in here - I was just wondering.